ui/eslint.config.js

```diff
@@ -12,26 +12,22 @@ const config = [
       'no-shadow': 'off',
       'no-unused-vars': 'off',
       'prefer-destructuring': 'off',
-      'import/no-named-as-default': 'off',
-      'import/no-named-as-default-member': 'off',
       'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/aria-role': 'off',
```

Comment:
Loosened in v1.0.5 to allow the use of a custom `role` prop on React components: https://gitlab.esrf.fr/ui/eslint-config/-/commit/7b39e158510b872ed8a99e31485ff77388c1b7ff